### DISCUSSION TITLE
A substitute for Fronius smart meter?

### DISCUSSION THREAD

**Discussion by kalaws** [goecharger/go-eController-API#5] [Q&A]
I’m considering getting the go-e controller, but it seems somewhat redundant to my current Fronius Smart Meter. They measure largely the same data, do they not? Would it be possible to supply the Fronius Datamanager with data from the go-e Controller, and simply remove the FSM?

**Comment by 0xFEEDC0DE64** [comment]
The go-eController has a SUNSPEC compatible modbus profile. Any systems accepting SUNSPEC smartmeter will happily accept a go-eController as their smartmeter!

**Comment by kalaws** [comment]
Plug and play?

Also, go-e controller can measure additional loads. Do you think/know if this functionality can be used with any sunspec inverter to feed data on secondary pv production coming from a another third party inverter? My guess is that the firmware of the first inverter (connected to goe controller) would somehow have to accommodate this?

**Comment by euro910** [comment]
anyone tried that already? 
i would like to use the controller with a symo as "smartmeter-ip" emulator, only wlan available to adapt Wallbox 14a and symo 60%
